src/app/components/settings/background/background.component.html

```diff
@@ -14,3 +14,5 @@ <h3 class="title">Change Background</h3>
 </div>
 <button class="trianglify btn btn-info" (click)="setTrianglify()" [disabled]="uploadingBackground">Use Randomly Generated Triangle Art for Background</button>
 <button class="btn btn-danger" (click)="deleteBackground()" [disabled]="hasDefaultBackground">Revert to Default Background</button>
+<h3 class="title" style="margin-top: 1rem">Layout</h3>
+<button class="btn btn-info" (click)="EditLayout()">Edit Home Page Layout</button>
```

Comment:
Because all you do is navigate to `/home/edit` with the `EditLayout()` function, you can condense that into `routerLink="/home/edit"` You can [read more about Angular routing here.](https://angular.io/guide/router)